src/pal/src/misc/time.cpp

```diff
@@ -28,8 +28,52 @@ Module Name:
 #include <errno.h>
 #include <string.h>
 
+#if HAVE_MACH_ABSOLUTE_TIME
+#include <mach/mach_time.h>
+static mach_timebase_info_data_t s_TimebaseInfo;
+#endif
+
 SET_DEFAULT_DEBUG_CHANNEL(MISC);
 
+static BOOL s_TimeInitialized = FALSE;
```

Comment:
I am new to the PAL source code, so I was trying to follow existing conventions. Example - https://github.com/dotnet/coreclr/blob/master/src/pal/src/exception/machexception.cpp#L74.

So I believe that if you are not freeing any resources in cleanup you dont need an 'init' variable.